File: BillPayments/Features/Billers/Data/Biller.swift
enum BillerMode: Equatable {
    case presentment
    case openAmount
}

struct BillerCategory: Equatable {
    let id: String
    let name: String
}

struct BillerInputParam: Equatable {
    let key: String
    let label: String
    let hint: String
}

struct Biller: Equatable {
    let id: String
    let categoryId: String
    let name: String
    let mode: BillerMode
    let inputParams: [BillerInputParam]
}

File: app/src/main/java/com/billpayments/features/billers/data/Biller.kt
package com.billpayments.features.billers.data

enum class BillerMode { Presentment, OpenAmount }

data class BillerCategory(val id: String, val name: String)

data class BillerInputParam(val key: String, val label: String, val hint: String)

data class Biller(
    val id: String,
    val categoryId: String,
    val name: String,
    val mode: BillerMode,
    val inputParams: List<BillerInputParam>,
)
